src/routes/navigation.js

```diff
@@ -0,0 +1,5 @@
+import { browserHistory } from '../root-reducer';
+
+export const goToRoute = (path) => {
+    browserHistory.push(path);
+};
```

Comment:
ok I was partially wrong with my comment above, I am used to seeing `<Redirect />` or `<Location />` but according to [the docs](https://reacttraining.com/react-router/web/api/location): 

```
// usually all you need
<Link to="/somewhere"/>

// but you can use a location instead
const location = {
  pathname: '/somewhere',
  state: { fromDashboard: true }
}

<Link to={location}/>
<Redirect to={location}/>
history.push(location)
history.replace(location)
```

So .... 
```
history.push(location)
history.replace(location)
```
Is part of the docs/recomended way.